web/gds-user-ui/src/modules/dashboard/organization/usePostOrganization.ts

```diff
@@ -1,7 +1,6 @@
-import { AxiosError } from 'axios';
 import { CreateOrganisation } from './organizationService';
 import { useMutation } from '@tanstack/react-query';
 
 export function usePostOrganizations() {
-  return useMutation<any, AxiosError>(CreateOrganisation);
+  return useMutation<any, any>(CreateOrganisation);
```

Comment:
While we're here can you change `CreateOrganisation` -> `CreateOrganization`? Might avoid some bugs in the future.